docs/application/dotnet/guides/nui/nui-components/Pagination.md

```diff
@@ -4,135 +4,222 @@ Pagination is a common component supporting multiple pages. It shows the number
+| `IndicatorColor`          | `Color`            | The indicator's color.                        |              
+| `SelectedIndicatorColor`  | `Color`            | The selected indicator's color.               |              
+| `SelectedIndex`           | `int`              | The index of the select indicator.            |              
+
+
+The base class for the Pagination is a [VisualView](https://docs.tizen.org/application/dotnet/guides/nui/visuals/), so it's properties can also be used, as showed in the examples below.
+
+
+
+<a name="createWithProperty"></a>
 ## Create with property
 
 To create a Pagination using property, follow these steps:
 
 1. Create Pagination using the default constructor:
 
     ```cs
-    pagination = new Pagination();
+    Pagination pagination = new Pagination();
     ```
 
-2. Set the Pagination property:
+2. Set the Pagination properties:
 
     ```cs
-    var indicatorImageUrlStyle = new PaginationStyle()
-    {
-        IndicatorImageUrl = new Selector<string>
-        {
-            Normal = "pagination_ic_nor.png",
-            Selected = "pagination_ic_sel.png"
-        }
-    };
-    pagination.ApplyStyle(indicatorImageUrlStyle);
-    pagination.IndicatorSize = new Size(26, 26);
-    pagination.IndicatorSpacing = 8;
-    pagination.Name = "Pagination1";
-    pagination.Size = new Size(300, 50);
-    pagination.BackgroundColor = new Color(1.0f, 1.0f, 1.0f, 0.6f);
-    pagination.IndicatorCount = 5;
-    pagination.SelectedIndex = 1;
-    root.Add(pagination);
-    ```
+   // path to the images 
+   string _imageUrl = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "images/";
+
+   // inherited properties of the pagination 
+   _pagination.Name = "Pagination1";
+   _pagination.Size = new Size(500, 150);
+   _pagination.BackgroundColor = new Color(0.4f, 0.56f, 1.0f, 0.7f);
+   _pagination.ParentOrigin = ParentOrigin.Center;
+   _pagination.PositionUsesPivotPoint = true;
+   _pagination.PivotPoint = PivotPoint.BottomCenter;
+
+   // specific properties of the Pagination
+   var _indicatorImageUrlStyle = new PaginationStyle()
+   {
+      IndicatorImageUrl = new Selector<string>
+      {
+         Normal = _imageUrl + "circle_unselected.png",
+         Selected = _imageUrl + "circle_selected.png"
+      }
+   };
+   _pagination.ApplyStyle(_indicatorImageUrlStyle);
+
+   _pagination.IndicatorSize = new Size(40, 40);
+   _pagination.IndicatorSpacing = 60;
+   _pagination.IndicatorCount = 5;
+   _pagination.SelectedIndex = 1;
+   ```
+
+   To set the absolute path of the used images the `Tizen.Applications.Application.Current.DirectoryInfo.Resource` was used
+   (for more information see
+   [Class Application](https://docs.tizen.org/application/dotnet/api/tizenfx/api/Tizen.Applications.Application.html)
+   and
+   [Class DirectoryInfo](https://docs.tizen.org/application/dotnet/api/tizenfx/api/Tizen.Applications.DirectoryInfo.html)).
```

Comment:
could you please help me with this one?
https://docs.tizen.org/application/dotnet/api/tizenfx/api/Tizen.NUI.BaseComponents.View.html

I'm not sure what the relative path is, since I can not find the source 
I checked in other NUI docs and there are used links to samsung.github, like:
https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html

Reply:
As we don't use URL starts with https://docs.tizen.org/... in guide pages, I left this comment without further checking.

In case of .NET API reference, we changed the URLs that refer https://docs.tizen.org to https://samsung.github.io.
So as you asked, for the .NET API reference URLs, please use URLs in https://samsung.github.io/.

I guess each URLs would be like below. Would you please confirm?
https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.Application.html
https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.DirectoryInfo.html

Reply:
Adding one more comment.
If you want to go to other **guide** page under the NUI, for example if you want to go to View, you can write in this way.

**old :** \[View](https://docs.tizen.org/application/dotnet/guides/nui/view/)
**Suggested :** \[View](../view.md)

Reply:
thank you very much !